### PR TITLE
Use doctrine/cs to ensure code standards of this project

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: DependencyInjection/Definition/CacheDefinition.php [hunk 1]
```diff
@@ -16,14 +16,10 @@
...
- *
- * @author Fabio B. Silva <fabio.bat.silva@gmail.com>
  */
 abstract class CacheDefinition
 {
-    /**
-     * @var string
-     */
+    /** @var string */
     private $type;
```
Comment (carusogabriel) [inline DependencyInjection/Definition/CacheDefinition.php]: Is safe to drop this write-only property in the class' constructor?

File: Command/DeleteCommand.php [hunk 2]
```diff
@@ -37,19 +39,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
 
         $success = $all ? $cacheProvider->deleteAll() : $cacheProvider->delete($cacheId);
         $color   = $success ? 'info' : 'error';
         $success = $success ? 'succeeded' : 'failed';
         $message = null;
 
-        if ( ! $all) {
+        if (! $all) {
             $message = "Deletion of <$color>%s</$color> in <$color>%s</$color> has <$color>%s</$color>";
             $message = sprintf($message, $cacheId, $cacheName, $success, true);
```
Comment (carusogabriel) [inline Command/DeleteCommand.php]: Is there a way to remove these variables inside the string on line 55 without affect this one?

Comment (jwage) [reply Command/DeleteCommand.php]: You would have to either move the variables down in to the `sprintf` or use string concatenation.

File: DependencyInjection/Definition/CacheDefinition.php (hunk 1 above)
Comment (jwage) [reply DependencyInjection/Definition/CacheDefinition.php]: Seems like it since the property is private and nothing else uses it.

**Comment by alcaeus** [comment]
@carusogabriel please don't be discouraged here, but with the potential deprecation of this bundle (see #156) I'd rather avoid doing too much cosmetic work just to throw it away.

**Comment by carusogabriel** [comment]
@alcaeus I have no problem in continue this one, but if this bundle is going to be deprecated, let's skip it 👍

**Comment by carusogabriel** [comment]
Closing as per @alcaeus' comment.

**Pull request closed without merging**